UI/CombatReport/GraphicalSummary.cpp

```diff
@@ -655,7 +655,7 @@ class OptionsBar : public GG::Wnd {
             button(nullptr)
         {
             button = Wnd::Create<CUIButton>("-");
-            parent->AttachChild(button);
+            parent_->AttachChild(button);
```

Comment:
@LGM-Doyle Should this be happening in `CompleteConstruction` ?

Reply:
This looks correct as is.

`ToggleData` is not  derived from `GG::Wnd`.  When `ToggleData` is created in `OptionsBar::CompleteConstruction()` both parent `OptionsBar` has a well defined `shared_from_this()`.